database/factories/ModelFactory.php

```diff
@@ -76,6 +78,13 @@
     ];
 });
 
+$factory->define(Setting::class, function ($faker) {
```

Comment:
One thing I could do is call the `cachet:seed` command(via the [Artisan facade](https://laravel.com/docs/5.3/artisan#programatically-executing-commands)) instead but I don't think it's going to be faster and it also creates data I don't need for my tests.